test/units/testsuite-77.sh

```diff
@@ -16,7 +16,7 @@ read -r -u 4 text
 assert_eq "$text" "File"
 
 # Test for socket
-systemctl start testsuite-77-netcat.service
+echo "Socket" | nc -lkU /tmp/test.sock
```

Comment:
won't this block?

Reply:
Yeah I just discovered that, any suggestions?

Reply:
systemd-run ?

Reply:
Then you get the race condition again, ugh how hard can it be to write a message to a socket and continue lol

Reply:
Still has a race condition without systemd-notify --ready, which we can't use because nc blocks

Reply:
I'll probably use a socket unit for this, will rework later